orchestration/dagster_orchestration/hca_manage/dataset.py

```diff
@@ -113,6 +114,13 @@ def _query_dataset(args: argparse.Namespace) -> None:
 class DatasetManager:
     environment: str
     data_repo_client: RepositoryApi
+    steward_list: list[str] = field(init=False)
+
+    def __post_init__(self) -> None:
+        self.steward_list = {
```

Comment:
⛏️ This is a dict, not a list. Perhaps name this `self.stewards`?

Reply:
Updated the name to be "stewards" and I think we are actually initializing it as a set, and later on the "add_policy_members" method takes in the policy_members parameter as a set[str] (thanks mypy)